src/icons/general/IconClock.js

```diff
@@ -1,22 +1,29 @@
 // @flow
 import * as React from 'react';
+import classNames from 'classnames';
 
 import AccessibleSVG from '../accessible-svg';
-
 import type { Icon } from '../flowTypes';
+import { bdlGray50 } from '../../styles/variables';
 
-const IconClock = ({ className = '', color = '#979797', height = 16, title, width = 16 }: Icon) => (
+const IconClock = ({ className = '', color = bdlGray50, height = 16, title, width = 16 }: Icon) => (
     <AccessibleSVG
-        className={`icon-clock ${className}`}
+        className={classNames('icon-clock', className)}
```

Comment:
Classname should follow SUIT style, see other new icon files as an example

Reply:
updated & updated associated scss